src/lib/consensus/intf.ml

```diff
@@ -608,8 +610,9 @@ module type S = sig
             { epoch_ledger : Mina_base.Epoch_ledger.Value.Stable.V1.t
             ; epoch_seed : Mina_base.Epoch_seed.Stable.V1.t
             ; epoch : Mina_numbers.Length.Stable.V1.t
-            ; global_slot : Mina_numbers.Global_slot.Stable.V1.t
-            ; global_slot_since_genesis : Mina_numbers.Global_slot.Stable.V1.t
+            ; global_slot : Mina_numbers.Global_slot_since_genesis.Stable.V1.t
```

Comment:
`Global_slot_since_hard_fork.t`- the difference between global_slot and global_slot_since_genesis in this type